pkg/auth/session.go

```diff
@@ -78,17 +80,17 @@ func (ss *SessionStore) pruneSessions() {
 			expired++
 		}
 	}
-	log.Debugf("Pruned %v expired sessions.", expired)
+	klog.V(4).Infof("Pruned %v expired sessions.", expired)
```

Comment:
We should make sure that the verbosity level aligns with the one we are setting int he console-operator [PR](https://github.com/openshift/console-operator/pull/250/files)